### DISCUSSION TITLE
XML & SOAP support

### DISCUSSION THREAD

**Discussion by ScriptKiddie47** [Netflix/conductor#3517] [Q&A]
My Org still has a lot of Rest services utilizing XML body & SOAP web services. I was wondering if Netflix conductor supports that.
I have searched around quite a bit , yet to find anything useful. Would it be even possible to re-engineer the conductor to support the above .Is the effort even worth it ?

**Comment by v1r3n** [comment]
Hi @ScriptKiddie47 its possible to use Conductor to work with XML and SOAP without re-engineering given the architecture of Conductor.  We can [chat](https://join.slack.com/t/orkes-conductor/shared_invite/zt-xyxqyseb-YZ3hwwAgHJH97bsrYRnSZg) more if you like, on how to achieve this with little effort.

**Reply by v1r3n** [reply]
I suggest take a look at this workflow https://play.orkes.io/execution/dedd6ca4-1f0c-4385-972a-629e0e84a986
There is a task `get_rate` that makes an XML request and `get_cost` task takes the XML to find the postage rate.
